inst/templates/validation_report.Rmd

```diff
@@ -98,8 +99,13 @@ vt_scrape_val_env() %>%
 #   vt_kable_functions()
 
 # include only exported function details
-all_func_details <- vt_scrape_functions()
-all_func_details[!is.na(all_function_details$export), c("functions", "editor", "editDate")] %>% 
-  vt_kable_functions()
-  
+# all_func_details <- vt_scrape_functions()
+# all_func_details[!is.na(all_function_details$export), c("functions", "editor", "editDate")] %>% 
+#   vt_kable_functions()
+#   
+```
+
+```{r child-files}
+child_files <- vt_get_child_files()
+quiet <- sapply(file.path(vt_path(), child_files), vt_file, dynamic_referencing = TRUE)
```

Comment:
should we make vt_file() vectorized? 🤔

Reply:
where else is `vt_file` being used? what would be affected?

Reply:
`vt_file` is only used internally here afaik. And if it's vectorized, all current single applications would still work i think.

All that needs to be done is change the current vt_file to vf_file_internal, and wrap it in a Vectorize and assign it to vt_file i believe (does that make sense?)